src/style.css

```diff
@@ -1 +1,46 @@
 /* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap");
+
+.header {
+  width: 100%;
+  height: 60px;
```

Comment:
don't set exact height for header, it should be calculated according to the content
just set correct width and height for the logo, and set correct top and bottom paddings for the header, so the full height will be also 60px